Automattic/wp-calypso
# A4A-Partner Directory: Highlight invalid form fields

Resolves https://github.com/Automattic/automattic-for-agencies-dev/issues/716

## Proposed Changes

![image](https://github.com/Automattic/wp-calypso/assets/9832440/a044303e-e601-45da-8940-b4fd8dd091c2)

This PR highlights the invalid field values. In this case, we check/validate the URLs and the email.

The validator highlight text will be displayed if you type at least three (3) characters. 

**Note**

In a follow-up PR, the Agency Details form could be saved at any moment without completing the whole form, even with no valid fields.
- In another PR, on each field update, the details form will be saved.

## Why are these changes being made?


*

## Testing Instructions

- Check the code
- Try to introduce an invalid URL (Any form field for URL, in the Details and Expertise form)
- Try to introduce an invalid Email (Company email, in the Details form)

## Pre-merge Checklist
